src/views/objectBrowser.js

```diff
@@ -1162,6 +1162,7 @@ class Member extends vscode.TreeItem {
     this.description = member.text;    
     this.resourceUri = getMemberUri(member, {filter: filter.name});
     this.path = this.resourceUri.path;
+    this.tooltip = `${this.resourceUri.path}\n(${member.text})`
```

Comment:
```suggestion
    this.tooltip = `${this.resourceUri.path}${member.text ? `\n(${member.text})` : ``}`;
```
I'd suggest to only add the text if it's present 😉

Reply:
Agree...